x-pack/plugins/alerts/server/authorization/alerts_authorization.test.ts

```diff
@@ -627,11 +626,17 @@ describe('AlertsAuthorization', () => {
       });
       alertTypeRegistry.list.mockReturnValue(setOfAlertTypes);
 
-      expect((await alertAuthorization.getFindAuthorizationFilter()).filter).toEqual(
-        esKuery.fromKueryExpression(
-          `((alert.attributes.alertTypeId:myAppAlertType and alert.attributes.consumer:(alerts or myApp or myOtherApp or myAppWithSubFeature)) or (alert.attributes.alertTypeId:myOtherAppAlertType and alert.attributes.consumer:(alerts or myApp or myOtherApp or myAppWithSubFeature)) or (alert.attributes.alertTypeId:mySecondAppAlertType and alert.attributes.consumer:(alerts or myApp or myOtherApp or myAppWithSubFeature)))`
-        )
-      );
+      // TODO: once issue https://github.com/elastic/kibana/issues/89473 is
```

Comment:
There are three test cases which used to compare `node_builder` output with KQL PEG parser output.  Now they're different, so to make sure that at least the `node_builder` output doesn't change over time, we'll do a snapshot compare.

I typically use inline snapshots, but in this case, since I didn't want to pollute the source files, I did snapshot files.  Nice experience actually!  

If the KQL PEG parser ends up getting fixed (see referenced issue), we can change these tests back to the original version.